apps/hash-api/src/graphql/resolvers/index.ts

```diff
@@ -111,6 +112,7 @@ export const resolvers: Omit<Resolvers, "Query" | "Mutation"> & {
     pageComments: loggedInAndSignedUpMiddleware(pageCommentsResolver),
     blocks: loggedInAndSignedUpMiddleware(blocksResolver),
     getEntity: getEntityResolver,
+    getFlowRuns,
```

Comment:
You're likely aware, but just realised we shouldn't merge this PR as is because it would expose the inputs/outputs of flow runs to any non-authenticated user. Perhaps we should add some check on getting just the users workflow runs or something like that, and until then block any response in the resolver when running in `production`.